parser.y

```diff
@@ -6098,6 +6098,19 @@ SetDefaultRoleStmt:
 			UserList: $6.([]*auth.UserIdentity),
 		}
 	}
+| 	"ALTER" "USER" IfExists Username "DEFAULT" "ROLE" SetDefaultRoleOpt
+	{
+		user := make([]*auth.UserIdentity, 0, 1)
+		user = append(user, $4.(*auth.UserIdentity))
+		tmp := $7.(*ast.SetRoleStmt)
+		$$ = &ast.SetDefaultRoleStmt{
+        	SetRoleOpt: tmp.SetRoleOpt,
+        	RoleList: tmp.RoleList,
+        	IsAlter: true,
+        	IfExist: $3.(bool),
+        	UserList: user,
+        }
```

Comment:
Fix indentation of lines 6107–6112 (reindent with tabs).

Reply:
ok